docker-compose.yml

```diff
@@ -8,3 +8,5 @@ services:
     - .:/opt/pulpd
   mongo:
     image: "mongo:4.2.1"
+    ports:
+      - "27017:27017"
```

Comment:
To allow host machines to talk to mongo from _outside_ the Docker container. Gotta get my Compass, yo